torchtitan/utils.py

```diff
@@ -80,18 +86,11 @@ def manual_seed(
     torch.distributed.tensor._random.manual_seed(seed)
 
 
-def set_determinism(deterministic: bool, seed: Optional[int] = None) -> None:
+def set_determinism(deterministic: bool) -> None:
     """
-    Set Python, PyTorch, CUDA seeds and cudnn settings for reproducibility
+    Set Determinism flags for increased reproducibility with loss of performance.
     """
 
-    if seed is not None:
-        # CPU and GPU determinism
-        torch.manual_seed(seed)
-        # set deterministic cudnn algorithms
-        # set Python seed
-        os.environ["PYTHONHASHSEED"] = str(seed)
-
     if deterministic:
         logger.info(
             f"Deterministic training enabled (expect perf degradation). Using seed: {seed}"
```

Comment:
how about let's combine the two functions `set_determinism` and `manual_seed` into one -- `set_determinism` doesn't seem to make sense without specifying a seed.

Reply:
yea i guess it is kind of pointless to do the determinism part without the seed part.  i can combine them back